website/docs/style/typography/typography.md

```diff
@@ -5,10 +5,7 @@ tabName: Design
 
 @## Font
 
-We use two font families in our interface:
-
-1. The main font is [Inter](https://fonts.google.com/specimen/Inter?query=inter).
-2. The accent font is [Factor A](https://type.today/en/factor_a).
+We use the [Inter](https://fonts.google.com/specimen/Inter?query=inter) font family in our interfaces.
```

Comment:
Let's change to:
`We use the [Inter](https://fonts.google.com/specimen/Inter?query=inter) font family in our design system`

I try to change "our interfaces" construction to "our design system" construction in all our guides because of our open source position.